src/ray/protobuf/gcs_service.proto

```diff
@@ -565,7 +565,7 @@ message InternalKVPutRequest {
 
 message InternalKVPutReply {
   GcsStatus status = 1;
-  int32 added_num = 2;
+  bool added = 2;
```

Comment:
nit: Do you mind add some comment here? I usually find it hard to understand our protobuf w/o reading through the code